docs/section21/Rule21-18.md

```diff
@@ -0,0 +1,41 @@
+
+# Boiler - Rule 21-18  
+
+**Rule ID:** 21-18  
+**Rule Description:** Baseline shall have only one heating hot water plant.  
+**Rule Assertion:** B-RMR LEN(FluidLoop.type == "HEATING") == 1  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Check:**
+
+1. Rule is applicable if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `PLACEHOLDER`
+
+## Rule Logic:  
+
+- For each fluid loop in B_RMR: `for fluid_loop_b in B_RMR.ASHRAE229.fluid_loops:`
+
+  - Check if fluid loop type is heating, add to heating hot water loop count: `if fluid_loop_b.type == "HEATING": hhw_loop_count += 1`
+
+**Rule Assertion:**
+
+- Case 1: If the total number of heating hot water loop in B_RMR is 1: `if hhw_loop_count == 1: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Note:**
+
+1. I was not able to find the exact language in App. G to put under the "Appendix G Section Reference".
+2. Is this requirement for B_RMR or for each building under B_RMR?
```

Comment:
This is for the entire B-RMR, i.e., all buildings included in the B-RMR. I beleive your RDS reflects that.